Bottom: f0fcc7b19fbf193987fbde7705b8c1af3d83721f
Top:    c06588f68211f92d67fea6f41b8fa0351e7c813e
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-11 16:29:54 -0500

Move ansible role vars to role defaults

This allows other parts of the playbook i.e. host vars, command line the
ability to override these defaults when necessary.


---
diff --git a/roles/docker-build-base-image/vars/main.yml b/roles/docker-build-base-image/defaults/main.yml
similarity index 100%
rename from roles/docker-build-base-image/vars/main.yml
rename to roles/docker-build-base-image/defaults/main.yml
diff --git a/roles/docker-exim4/vars/main.yml b/roles/docker-exim4/defaults/main.yml
similarity index 100%
rename from roles/docker-exim4/vars/main.yml
rename to roles/docker-exim4/defaults/main.yml
diff --git a/roles/docker-openssl/vars/main.yml b/roles/docker-openssl/defaults/main.yml
similarity index 100%
rename from roles/docker-openssl/vars/main.yml
rename to roles/docker-openssl/defaults/main.yml
diff --git a/roles/docker-pull-images/vars/main.yml b/roles/docker-pull-images/defaults/main.yml
similarity index 100%
rename from roles/docker-pull-images/vars/main.yml
rename to roles/docker-pull-images/defaults/main.yml
